brainstormforce/ultimate-addons-for-gutenberg
# Fix: Added the WP 5.9 Responsive Dynamic Assets in Editor Code for 10 Blocks.

### Description

- https://docs.google.com/spreadsheets/d/1LcjCmQGmg36vPP53lFljlTYbMr7Urg2eOWrY7mOyVXU/edit#gid=0

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Test cases for Responsive CSS -
 - Add color or any style to block and check it in tablet/mobile preview.
 - Add a responsive CSS like font size on tablet/mobile and check if it applies in the editor or not.
 - Check style tag is loaded in an iframe when the preview is tablet/mobile

### Checklist:
